src/docfx/lib/JsonUtility.cs

```diff
@@ -408,59 +427,43 @@ protected override JsonProperty CreateProperty(MemberInfo member, MemberSerializ
                 return prop;
             }
 
-            private SchemaValidationConverter GetConverter(MemberInfo member)
+            private SchemaValidationAndTransformConverter GetConverter(MemberInfo member)
```

Comment:
nit: what if we just name them `SchemaConverter` and `SchemaContractResolver` ?

Reply:
Thought we are doing both inside, and since these are private... 

---
In reply to: [210854391](https://github.com/dotnet/docfx/pull/3209#discussion_r210854391) [](ancestors = 210854391)